src/server/page_server.cpp

```diff
@@ -283,31 +321,35 @@ void page_server::disconnect_all_tran_server ()
 			m_passive_tran_server_conn[i]->get_channel_id ().c_str ());
 	  m_passive_tran_server_conn[i].reset (nullptr);
 	}
-      m_passive_tran_server_conn.clear();
+      m_passive_tran_server_conn.clear ();
     }
 }
 
-bool page_server::is_active_tran_server_connected () const
+bool
+page_server::is_active_tran_server_connected () const
 {
   assert (is_page_server ());
 
   return m_active_tran_server_conn != nullptr;
 }
 
-bool page_server::is_passive_tran_server_connected () const
+bool
+page_server::is_passive_tran_server_connected () const
 {
   assert (is_page_server ());
 
   return !m_passive_tran_server_conn.empty ();
 }
 
-cublog::async_page_fetcher &page_server::get_page_fetcher ()
+cublog::async_page_fetcher *
```

Comment:
Why change from reference to pointer?

Reply:
Leftover from trying to lift `ps_Gl` from page_server.hpp to server_type.hpp. Which, by the way, can only be done if `ps_Gl` is also changed to a unique_ptr.
I'll revert this change for now.